lib/index.js

```diff
@@ -33,7 +33,9 @@ function swaggerize(server, options) {
  * @returns {onmount}
  */
 function mount(server, options) {
-    Object.defineProperty(server, 'swagger', {
+    if(!options.version)
+        options.version = 'v1';
+    Object.defineProperty(server, 'swagger'+options.version, {
```

Comment:
@bardzusny made swagger configurable and writable. This will allow to call swaggerize multiple times. 

Object.defineProperty(server, 'swagger', {
        configurable: true,
        writable: true